e2e-tests/portal/cypress/e2e/journeys/maker/submit-issued-facilities-for-review/submit-issued-bond-facility-with-created-field.spec.js

```diff
@@ -35,16 +50,22 @@ context('Issue Bond Form - Submit issued bond with inserted element on page', ()
   });
 
   it("should not insert created element's data into the bond", () => {
-    cy.login(BANK1_MAKER1);
-    pages.contract.visit(deal);
+    cy.loginGoToDealPage(BANK1_MAKER1);
+    pages.dashboardDeals.visit();
+    cy.clickDashboardDealLink();
+
     pages.contract.proceedToReview().should('not.exist');
 
     const bondId = dealFacilities.bonds[0]._id;
     const bondRow = pages.contract.bondTransactionsTable.row(bondId);
 
-    bondRow.issueFacilityLink().click();
+    bondRow.uniqueNumberLink().click();
+
+    pages.bondDetails.facilityStageIssuedInput().click();
 
-    cy.completeDateFormFields({ idPrefix: 'issuedDate' });
+    cy.keyboardInput(pages.bondDetails.requestedCoverStartDateDayInput(), startDate.day);
+    cy.keyboardInput(pages.bondDetails.requestedCoverStartDateMonthInput(), startDate.month);
+    cy.keyboardInput(pages.bondDetails.requestedCoverStartDateYearInput(), startDate.year);
```

Comment:
Merge issue here - should be the following:

```js
cy.completeDateFormFields({ idPrefix: 'issuedDate' });
```

Reply:
The test is not entering the cover start date when `cy.completeDateFormFields({ idPrefix: 'issuedDate' });` is used

Reply:
Sorry that was a typo - need to pass in the correct ID into `idPrefix`, not `issuedDate`.

Reply:
To be checked.

Reply:
Changed to idPrefix: 'requestedCoverStartDate'